src/backends/fixerio/index.js

```diff
@@ -32,9 +31,6 @@ class FixerIoBackend {
     this.rates = {}
     this.currencies = []
     const ledgerPairs = config.get('tradingPairs')
-    if (_.isEmpty(ledgerPairs)) {
```

Comment:
I meant that we should check for `null`/`undefined`, sorry I should have caught this in the review